packages/Frontend/src/components/Authenticate.jsx

```diff
@@ -15,6 +15,21 @@ const StyledDiv = styled.div`
   width: 50%;
 `;
 
+const handleErrorMessage = (error) => {
```

Comment:
Firebase gives us very predictable error codes with everything, which is awesome, amazon seemed very inconsistent with theirs. You can see them here in their documentation: https://firebase.google.com/docs/reference/js/firebase.auth.Auth#createUserWithEmailAndPassword. This function takes the error codes we're given and if there's a match returns a string that we actually show the user.

They have two different types of documentation, guides and reference, this is their reference which may look a little scary until you get used to it. Their guides are really good though. Here's the guide I followed to set up a user with an email address and password: https://firebase.google.com/docs/auth/web/password-auth

Reply:
Also, switch case is a feature in Javascript that I would probably shy away from most of the time, it's easy to make mistakes with it. However, this is a pretty good use case for it. MDN has a page on how it works you can experiment with: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch